Assets/Scripts/Tank/IDamageable.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using System;
 public interface IDamageable
 {
-	void TakeDamage(float damage);
+	void TakeDamage(float damage,int id);
```

Comment:
id for what ? 
tank id? bullet id? something else id? not clear from the variable name